src/components/data-management/ProjectDetails.jsx

```diff
@@ -1,31 +1,68 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
 /* eslint-disable react/jsx-props-no-spreading */
-import React, { useRef } from 'react';
+import React, { useRef, useState, useMemo } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 
 import {
-  Space, Col, Row, Typography, Button,
+  Space, Typography, Button,
 } from 'antd';
 import PropTypes from 'prop-types';
-import SamplesTable from './SamplesTable';
-import ProjectMenu from './ProjectMenu';
 import {
   updateProject,
-} from '../../redux/actions/projects';
+} from 'redux/actions/projects';
+import { layout } from 'utils/constants';
+import SamplesTable from './SamplesTable';
+import ProjectMenu from './ProjectMenu';
 
-const { Title, Text, Paragraph } = Typography;
+const {
+  Title, Text, Paragraph, Link,
+} = Typography;
 
 const ProjectDetails = ({ width, height }) => {
+  const { PANEL_HEADING_HEIGHT, PANEL_PADDING } = layout;
+
+  const availableWidth = width - PANEL_PADDING * 2; // * 2 because left and right
+  const availableHeight = height - PANEL_HEADING_HEIGHT - PANEL_PADDING * 2; // * 2 because top and bottom
```

Comment:
We still need this because `width` and `height` as passed by `react-resize-observer` is the width and the height of the panel component, which includes the panel title (30 px) and the panel's padding (10px). So, we have to calculate the available width and height and properly pass it on to the elements that need them.

Reply:
Removed as the code is rewritten using flex